wiki/Layout-component/Layout.md

```diff
@@ -39,4 +37,128 @@ The Layout core component helps us to layout our different components with a mor
 
 <br />
 
-> ![image](https://user-images.githubusercontent.com/3793636/117873919-f6faba80-b265-11eb-81a5-039bdcd822e8.png)  See box folder <sup>[[r1](#resources)]</sup> for more details
\ No newline at end of file
+### 1. Type 
+
+`required` `string`
+
+The different types allows an adopter to choose an approved layout option. The different types will layout an adopters content in different ways, and responsively behave
+
+|      | type   | description | visual |
+|:-----|:-------|:------------|:-------|
+| 1.1  | `4-12` | Full 16 column layout: 4 columns left, 12 columns right. | <a href="https://user-images.githubusercontent.com/3793636/121963487-c9ef6b00-cd2f-11eb-9845-8f8a17bb44d1.png" target="_blank"><img src="https://user-images.githubusercontent.com/3793636/121963487-c9ef6b00-cd2f-11eb-9845-8f8a17bb44d1.png" alt="4-12" width="300px" /></a>  |
+| 1.2  | `8-4`  | Intended to be nested within 12 columns. | <a href="https://user-images.githubusercontent.com/3793636/121963489-ca880180-cd2f-11eb-9ba3-ee893e03d820.png" target="_blank"><img src="https://user-images.githubusercontent.com/3793636/121963489-ca880180-cd2f-11eb-9ba3-ee893e03d820.png" alt="8-4" width="300px" /></a>  |
+
+
+<br />[Back to top](#wiki-wrapper)<br /><br /><br />
+
+
+
+### 2. Children 
+
+`required` `container`
+
+The children property allows the adopter to place their content within the correct column to be laid out. The direct child `HTML`/`JSX` provided gets the column styling. For example if you choose a 2 column layout you will need 2 child wrappers, or 3 columns you will need 3 child wrappers. See the code examples below.
+
+###### Examples
+
+![layout](https://user-images.githubusercontent.com/3793636/122071996-d372e480-cdbc-11eb-8242-38eaee8d583f.gif)
+
+```html
+<!-- Here is an example where you’re using **two** columns. -->
+
+<Layout type=“4-12”>
+  <div>{content for column 1}</div>
+  <div>{content for column 2}</div>
+</Layout>
+
+<!-- Here is an example where you might have **four** columns. -->
+
+<Layout type=“4-4-4-4”>
+  <div>{content for column 1}</div>
+  <div>{content for column 2}</div>
+</Layout>
+
+<!-->
+    Note that `4-4-4-4` layout is not an option at this time.
+
+    Also, notice how each column has a corresponding child.
+    You could also optionally leave off a column from right
+    to left if you don’t need it for a given layout.
+ </-->
+```
+
+> ![image](https://user-images.githubusercontent.com/3793636/117873641-a6835d00-b265-11eb-8433-8c9c73a2e999.png) Note the code examples above maybe react specific, but the functionality should stand across any language with how it relates to being built out.
```

Comment:
As a non-dev I might suggest changing "across any language" to "across any framework", when I first read it I thought of language = translating to different geographies.

```suggestion
> ![image](https://user-images.githubusercontent.com/3793636/117873641-a6835d00-b265-11eb-8433-8c9c73a2e999.png) Note the code examples above maybe react specific, but the functionality should stand across any framework with how it relates to being built out.
```

Reply:
I just removed the whole thing. Confusing, and blurring the line between technical documentation.